composer.json

```diff
@@ -7,7 +7,9 @@
     "license": "Apache-2.0",
     "require": {
         "php": "^7.4 || ^8.0",
+        "php-64bit": "^7.4",
```

Comment:
I think this replaces the `php` entry above

Reply:
Interesting, kinda weird it let me do that...got rid of that line and updated the existing one to say php-64bit